src/lib.rs

```diff
@@ -1029,6 +1043,22 @@ impl Instance {
             })
     }
 
+    /// Retrieves an [adapter][Adapter] which matches the given [options][RequestAdapterOptions].
+    ///
+    /// Some options are "soft", so treated as non-mandatory. Others are "hard".
+    ///
+    /// If no adapters are found that suffice all the "hard" options, `None` is returned.
+    pub fn request_adapter(
```

Comment:
I've moved this function to change the order it shows up in the generated documentation; now both adapter-related functions are together.